src/modules/utils/panel/screens/FileScreen.cpp

```diff
@@ -69,20 +71,43 @@ void FileScreen::enter_folder(const char *folder)
     this->refresh_menu();
 }
 
-// Called by the panel when refreshing the menu, display .. then all files in the current dir
+
 void FileScreen::display_menu_line(uint16_t line)
```

Comment:
should remove this if it is not used, and remove the pure virtual in panelscreen (remove the =0)